src/types.h

```diff
@@ -102,6 +100,14 @@ class SizedType
   bool IsEqual(const SizedType &t) const;
   bool operator==(const SizedType &t) const;
   bool operator!=(const SizedType &t) const;
+
+  bool IsSigned(void) const;
+
+  friend std::ostream &operator<<(std::ostream &, const SizedType &);
+  friend std::ostream &operator<<(std::ostream &, Type);
+
+private:
+  bool is_signed = false;
```

Comment:
Style nit
```suggestion
  bool is_signed_ = false;
```

Reply:
I'll sneak this in with the follow up :)